.github/workflows/main.yml

```diff
@@ -11,20 +11,19 @@ jobs:
         include:
           - os: windows-latest
             triplet: x64-windows
-            vcpkgGitCommitId: '6282cab61d175974b1af06473db584b9b80dcd48'
-            vcpkgPackages: 'grpc'
           - os: ubuntu-latest
             triplet: x64-linux
```

Comment:
Remove triplet

Reply:
@magnesj: `triplet` is being used n two places to create the env var `vcpkgResponseFile`. Probably I can see that i could use set-env and set it once in the strategy's  matrix, that is:

  ```
include:
  - os: ubuntu-latest
    vcpkg_response_file: '${{ github.workspace }}/vcpkg_x64-linux.txt'
```

Reply:
Ok, this adjustment can be postponed. Merging now.